packages/api-client/src/main/auth/RegisterData.ts

```diff
@@ -17,11 +17,12 @@
  *
  */
 
+import {UserAccentColor} from '@wireapp/commons/dist/commonjs/util/AccentColor';
```

Comment:
Is it possible to have an import in the form of `import {Something} from '@wireapp/commons` here?

Reply:
How do you import `AccentColor` at the moment in other projects?